### PR TITLE
[stable/minio] Refs #81 - fix stable versioning in KubeVersion, specifically v 1.16

### PR DESCRIPTION
**Pull request by patvdleer**
Refs #81 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/minio]`)

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -3,7 +3,27 @@ entries:
...
+      name: MinIO, Inc
+    name: minio
+    sources:
+    - https://github.com/minio/minio
+    urls:
+    - https://helm.min.io/releases/minio-8.0.7.tgz
+    version: 8.0.7
+  - apiVersion: v1
+    appVersion: master
+    created: "2020-12-07T15:49:39.954547328+01:00"
```
Comment (harshavardhana) [inline index.yaml:26]: Community is not allowed to do this @patvdleer you should refrain from doing this yourself.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: index.yaml (hunk 1 above)
Comment (patvdleer) [reply index.yaml]: index reverted

**Review by harshavardhana — CHANGES_REQUESTED** [review]
You may have to revert all the changes in index.yaml @patvdleer `git reset HEAD^ index.yaml; git checkout index.yaml; git commit --amend`

**Comment by patvdleer** [comment]
I did `git checkout 2f108ca -- index.yaml` 2f108ca being v8.0.6

**Comment by patvdleer** [comment]
completely removed the change to `index.yaml` and reapplied the version bump

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**